source/reference/minio-server/settings/notifications.rst

```diff
@@ -0,0 +1,71 @@
+.. _minio-server-envvar-notifications:
+
+=================================
+Settings for Bucket Notifications
```

Comment:
We're L4 deep in here.

I wonder if we should drop the second level 'Settings" bucket here and do something like

```
/minio-server/minio-server.rst
/minio-server/configure-core-settings.rst
/minio-server/configure-identity-access-management.rst
/minio-server/configure-bucket-notifications.rst
```

It's a pseudo-prefix, but gets us into at max an L3 left nav.

Reply:
I didn't improve the navigation on this pass. I can truncate things some if this remains a concern.

Reply:
Oh, never mind, I did get it done.